app/build.gradle.kts

```
@@ -22,10 +22,7 @@ android {
 
     signingConfigs {
         getByName("debug") {
-            keyAlias = project.property("keyAlias") as String
-            keyPassword = project.property("keyPassword") as String
-            storeFile = rootProject.file(project.property("storeFile") as String)
-            storePassword = project.property("storePassword") as String
+
```

@annshsingh Can you revert these changes as it will break on CICD.
```suggestion
            keyAlias = project.property("keyAlias") as String
            keyPassword = project.property("keyPassword") as String
            storeFile = rootProject.file(project.property("storeFile") as String)
            storePassword = project.property("storePassword") as String
```

---

Okay sure.